PhysicsTools/NanoAOD/python/custom_jme_cff.py

```diff
@@ -364,7 +368,6 @@ def AddBTaggingScores(proc, jetTableName=""):
   """
 
   getattr(proc, jetTableName).variables.btagDeepB       = BTAGVARS.btagDeepB
-  getattr(proc, jetTableName).variables.btagCSVV2       = BTAGVARS.btagCSVV2
```

Comment:
Shouldn't these be put back in for `run2_nanoAOD_ANY` as well?

Reply:
I will check a run2 wf and add it back in case.
Should I add the modifier inside this function?

Reply:
I think the `BTAGVARS` PSet can be modified directly (e.g. in L175)?

Reply:
Indeed, but in some cases a new table is created instead of reusing the one from default NanoAODs, e.g. in https://github.com/cms-sw/cmssw/blob/35422366701d25c18fc6b8c4d1bf8459979b2c39/PhysicsTools/NanoAOD/python/custom_jme_cff.py#L498 ?
Haven't checked when that is done in practice, perhaps @nurfikri89 can chime in...

Reply:
Ok got it, thanks!
Indeed I was testing a few workflows from the matrix that run the `custom_jme_cff` (both for Run2 and Run3) and none of them invokes this method to add the btagging info to the table! 😄 

But I agree it should be there just in case: so I will add this customization in the function!

Reply:
I added in the tests also the `2500.31` wf which actually runs the `run2_nanoAOD_106Xv2` era.